jekyll/_cci2/deployment-integrations.md

```diff
@@ -16,46 +14,60 @@ CircleCI can be configured to deploy to virtually any service. This document pro
 ## Overview
 {:.no_toc}
 
-To deploy your application, add a [job]({{ site.baseurl }}/2.0/jobs-steps/#jobs-overview) to `.circleci/config.yml`. If your deploy job uses any output from previous jobs, you can share that data by [using workspaces]({{ site.baseurl }}/2.0/workflows/#using-workspaces-to-share-data-among-jobs). You will also have to set any necessary secrets on the **Project Settings > Environment Variables** page of the CircleCI application.
+To deploy your application, add a [job]({{ site.baseurl }}/2.0/jobs-steps/#jobs-overview) to `.circleci/config.yml`. You will also have to set any necessary secrets on the **Project Settings > Environment Variables** page of the CircleCI application.
 
-Deploying is usually dependent on a successful build. The following example uses a [workflow job filter]({{ site.baseurl }}/2.0/workflows/#using-contexts-and-filtering-in-your-workflows) to check that the `master` branch is checked out before running any deploy commands. Without this filter, deploy commands would be executed every time this job is triggered.
+Below is a simple example of deploying a Rails application to Heroku. The configuration uses [workflows]({{ site.baseurl }}/2.0/workflows/) to deploy only if the `sequential-branch-filter` branch is checked out and the `build` job has run.
```

Comment:
Does this duplicate what is in the heroku section? I think having a very basic example that is generic first, followed by sections specific to each target makes sense, not sure why you are reworking it (they all use workflows already)?

Reply:
There's some duplication, but this is actually a complete configuration example. The weakness with the generic deploying example was that it was _too_ generic. The issuer-filer wasn't able to get a working config by using it as a template.

The Heroku example, on the other hand, is too _specific_ and doesn't present a complete configuration example. I think there may be more value in seeing a complete example of a working deploy first, followed by a discussion of platform-specific nuances.